### PR TITLE
vendor: Bump cAdvisor to v0.43.0

### PR DESCRIPTION
**Pull request by bobbypage**
#### What type of PR is this?

/kind bug

#### What this PR does / why we need it:

Bump cAdvisor to v0.43.0. 

Changes: https://github.com/google/cadvisor/compare/v0.39.2...v0.43.0 

and cAdvisor release notes have PRs included: https://github.com/google/cadvisor/releases

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: vendor/modules.txt [hunk 1]
```diff
@@ -179,18 +179,14 @@ github.com/container-storage-interface/spec/lib/go/csi
...
 # github.com/containerd/console v1.0.2 => github.com/containerd/console v1.0.2
 github.com/containerd/console
-# github.com/containerd/containerd v1.4.9 => github.com/containerd/containerd v1.4.9
+# github.com/containerd/containerd v1.4.11 => github.com/containerd/containerd v1.4.11
 github.com/containerd/containerd/api/services/containers/v1
 github.com/containerd/containerd/api/services/tasks/v1
 github.com/containerd/containerd/api/services/version/v1
 github.com/containerd/containerd/api/types
 github.com/containerd/containerd/api/types/task
-github.com/containerd/containerd/containers
```
Comment (dims) [inline vendor/modules.txt:188]: nice set of cleanups! yay. the rest of `github.com/containerd/containerd` will get cleaned out when we remove dockershim for sure. (or the change in moby/moby we are waiting on)

**Review by dims — APPROVED** [review]
/approve
/lgtm

**Review by harche — APPROVED** [review]
/lgtm

**Review by SergeyKanzhelev — APPROVED** [review]
I'm curious if any of bug fixes worth mentioning in release notes for k8s. I see it was like this without release notes most of the time

/lgtm

**Comment by liggitt** [comment]
/approve

for deps

**Comment by vsxen** [comment]
backport to v1.20? @bobbypage 

https://github.com/containerd/containerd/issues/4541
https://github.com/google/cadvisor/pull/2956

**Comment by pacoxu** [comment]
> backport to v1.20? @bobbypage
> 
> [containerd/containerd#4541](https://github.com/containerd/containerd/issues/4541) [google/cadvisor#2956](https://github.com/google/cadvisor/pull/2956)

@dims  @liggitt @SergeyKanzhelev  Should this be backported to 1.19-1.22 as the bump included bug fixes for metrics?

**Comment by pacoxu** [comment]
> backport to v1.20? @bobbypage
> 
> [containerd/containerd#4541](https://github.com/containerd/containerd/issues/4541) [google/cadvisor#2956](https://github.com/google/cadvisor/pull/2956)

@dims  @liggitt @SergeyKanzhelev  Should this be backported to 1.19-1.22 as the bump included bug fixes for metrics?

**Pull request merged**